Libraries/oneVPL/hello-decode/README.md

```diff
@@ -1,11 +1,11 @@
 # `hello-decode` Sample
 
 This sample shows how to use the oneAPI Video Processing Library (oneVPL) to
-perform a simple video decode.
+perform a simple video decode using 2.x enhanced programming model APIs.
 
 | Optimized for    | Description
 |----------------- | ----------------------------------------
-| OS               | Ubuntu* 18.04; Windows* 10
+| OS               | Ubuntu* 20.04; Windows* 10
```

Comment:
Most samples work in 18.04 and 20.04, is there a reason for the change?

Reply:
Only 20.04 is supported for GPU.